apps/backend/src/main/resources/api.yaml

```diff
@@ -13,4 +13,108 @@ paths:
     get:
       responses:
         400:
-          description: Bad request
\ No newline at end of file
+          description: Bad request
+components:
+  schemas:
+    ModelDTO:
+      required:
+        - description
+        - name
+      properties:
+        id:
+          type: integer
+        name:
+          type: string
+        description:
+          type: string
+    LocationDTO:
+      required:
+        - description
+        - name
+        - parentLocationId
+      properties:
+        id:
+          type: integer
+        name:
+          type: string
+        description:
+          type: string
+        parentLocationId:
+          type: integer
+          nullable: true
+    ItemDTO:
+      required:
+        - modelId
+        - movementsId
+      properties:
+        id:
+          type: integer
+        modelId:
+          type: integer
+        movementsId:
+          type: array
+          items:
+            type: integer
+    MovementDTO:
+      required:
+        - locationId
+        - itemId
+        - movementType
+      properties:
+        id:
+          type: integer
+        locationId:
+          type: integer
+        itemId:
+          type: integer
+        movementType:
+          type: string
+    UserDTO:
+      required:
+        - email
+        - firstName
+        - lastName
+        - username
+      properties:
+        id:
+          type: integer
+        username:
+          type: string
+        email:
+          type: string
+        firstName:
+          type: string
+        lastName:
+          type: string
+    InventoryDTO:
+      required:
+        - ownerId
+        - participants
+      properties:
+        id:
+          type: integer
+        ownerId:
+          type: integer
+        participants:
+          type: array
+          items:
+            type: integer
+    ItemMetaDataDTO:
+      required:
+        - itemId
+        - name
+        - content
+      properties:
+        itemId:
+          type: string
+        name:
+          type: string
+        content:
+          type: string
```

Comment:
It does not seem necessary to add this on the current scope.

It can stay, but won't probably used.